src/content/docs/en/pages/main-menu/reference/store/edge-sql/edge-sql.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Edge SQL
+description: Explore Edge SQL, a distributed database solution by Azion. Learn to manage databases, run queries, and ensure data resilience effortlessly.
+meta_tags: edge sql, sql, database, distributed, queries
+namespace: docs_edge_sql
+permalink: /documentation/products/store/edge-sql/
+---
+
+import Badge from '~/components/Badge.astro'
+import Button from '~/components/Button.astro'
+
+<Badge variant="accent">
+  Preview
+</Badge>
+
+Azion **Edge SQL** is an edge-native SQL solution designed for serverless applications. It's fully ACID-compliant and utilizes SQLite's SQL dialect to provide a familiar development environment, facilitating fast integration. Azion Edge SQL implements a Main/Replicas architecture to enable ultra-low latency querying at the network edge, enhancing performance for distributed applications.
+
+With Edge SQL, you can:
+
+- Store real-time telemetry data, ideal for IoT.
+- Manage inventory of distributed e-commerce/retail distribution centers.
+- Store and analyze access logs, quickly identifying and mitigating security attacks.
+- Store and manage access data.
+
+There are two ways to build a database through Edge SQL: using the Edge SQL REST API or writing an edge function. The REST API allows you create, read, update, delete, and list databases. An edge function allows you to write on a database.
+
+### Implementation
+
+| Scope | Resource |
+| ----- | -------- |
+| Manage databases with REST API | [How to create an Edge SQL database](/en/documentation/products/guides/manage-databases-edge-sql/) |
+| Create queries with REST API | [How to create and query data on Edge SQL](/en/documentation/products/guides/create-tables-edge-sql/) |
+| Write on a database with an edge function | []() |
+
+---
+
+## Business rules
+
+Writing databases, through REST API or edge function, is only possible in the [main instance](#main-instance). All [replicas](#replicas) are read-only.
+
+:::caution[warning]
+Once you delete a main instance, this action can't be undone. You'll no longer be able to write or read on that database.
+:::
+
+During the creation of a database, especially your first one, there's a propagation time for it to be created on the edge nodes. Your `POST` request response will exhibit a `creating` state, and you can run `GET` requests to check once it's created.
```

Comment:
for it to be replicated on the edge nodes OR  for it to be propagated on the edge nodes ?